test/e2e_cri/container.go

```diff
@@ -0,0 +1,145 @@
+/*
+Copyright 2016 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package e2e_cri
+
+import (
+	internalApi "k8s.io/kubernetes/pkg/kubelet/api"
+	runtimeApi "k8s.io/kubernetes/pkg/kubelet/api/v1alpha1/runtime"
+	"k8s.io/kubernetes/pkg/util/uuid"
+	"k8s.io/kubernetes/test/e2e/framework"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+var (
+	imageName = "busybox"
+)
+
+func testCreateContainer(c internalApi.RuntimeService) (string, string) {
+	By("create a podSandbox with name")
+	podName := "sandbox-for-create-container-" + string(uuid.NewUUID())
+	podConfig := &runtimeApi.PodSandboxConfig{
+		Metadata: &runtimeApi.PodSandboxMetadata{Name: &podName},
+	}
+	podID, err := c.RunPodSandbox(podConfig)
+	framework.ExpectNoError(err, "Failed to create podsandbox: %v", err)
+	framework.Logf("Created Podsanbox %s\n", podID)
+
+	By("create container")
+	containerName := "container-for-create-test-" + string(uuid.NewUUID())
+	containerConfig := &runtimeApi.ContainerConfig{
+		Metadata: &runtimeApi.ContainerMetadata{Name: &containerName},
```

Comment:
CRI assumes all fields in ContainerMetadata are set by kubelet, maybe extract a helper for default metadata?